.gitignore

```diff
@@ -1,2 +1,2 @@
-/bin/
-/pkg/
+vendor/
+*.tar.xz
```

Comment:
the toplevel `abireport` binary should be ignored as well

Reply:
Indeed, changing.